addon/locales/en/translations.js

```diff
@@ -223,18 +223,24 @@ export default {
                 enterComponentTitle: 'Enter component title',
                 newProjectMessage: 'You have selected to create a new public project for your file. Users will still have access to your file unless the project becomes private.',
                 chooseProject: 'Choose project',
-                projectSelectMessage: 'The list of projects appearing in the selector are projects and components for which you have admin access. Registrations are not included here.',
+                projectSelectMessage: 'The list of projects appearing are projects and components for which you have write access. Registrations are not included here.',
                 noProjectsExistError: 'You have no available projects. Go back to create a new project.',
                 organizeProject: 'Organize',
-                organizeParentNode: 'You can organize your file by storing it in the project or in its own component. If you select "Make a new component", the file will be stored in a new component inside the project. If you select "Use the current component", the file will be stored inside the project. If you are unsure, select "Make a new component".',
-                organizeChildNode: 'You can organize your file by storing it in this component or in its own component. If you select "Make a new component", the file will be stored in a new component inside this component. If you select "Use the current component", the file will be stored inside this component. If you are unusure, select "Make a new component".',
+                organizeParentNode: 'You can organize your file by storing it in the project or its own component.',
+                organizeChildNode: 'You can organize your file by storing it in the component or its own component.',
```

Comment:
Hmm, not sure about s/this/the/ here. The requested changes just said to basically s/project/component/, but maybe this > the here?

Reply:
I agree, this language is confusing.

Reply:
Yes, it is, but leaving as-is for now.